apis/core/v1alpha2/function_conversion.go

```diff
@@ -141,13 +141,14 @@ func (src *Function) convertServingTo(dst *v1beta1.Function) error {
 			if src.Spec.Serving.OpenFuncAsync.Dapr.Inputs != nil {
 				dst.Spec.Serving.Inputs = []*v1beta1.DaprIO{}
 				for _, input := range src.Spec.Serving.OpenFuncAsync.Dapr.Inputs {
+					i := input.DeepCopy()
 					in := &v1beta1.DaprIO{
```

Comment:
in is not necessary after i:= input.Deepcopy()?
we can use i directly?

Reply:
I don't think we can use `i` directly because it is a type `v1alpha2.DaprIO` object...

We can use the value of `input` directly:

```go
        for _, input := range src.Spec.Serving.OpenFuncAsync.Dapr.Inputs {
          in := &v1beta1.DaprIO{
            Name:      input.Name,
            Component: input.Component,
            Type:      input.Type,
            Topic:     input.Topic,
            Params:    input.Params,
            Operation: input.Operation,
          }
          dst.Spec.Serving.Inputs = append(dst.Spec.Serving.Inputs, in)
        }
```

Reply:
ok, agree